app/src/main/java/edu/outside2154/gamesense/fragment/ChecklistFragment.kt

```diff
@@ -0,0 +1,94 @@
+package edu.outside2154.gamesense.fragment
+
+import android.content.Context
+import android.net.Uri
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+import edu.outside2154.gamesense.R
+
+/**
+ * A simple [Fragment] subclass.
+ * Activities that contain this fragment must implement the
+ * [ChecklistFragment.OnFragmentInteractionListener] interface
+ * to handle interaction events.
+ * Use the [ChecklistFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class ChecklistFragment : Fragment() {
+
+    // TODO: Rename and change types of parameters
+    private var mParam1: String? = null
+    private var mParam2: String? = null
+
+    private var mListener: OnFragmentInteractionListener? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        if (arguments != null) {
```

Comment:
Optional: below is more Kotlin-ish.

```kotlin
arguments?.run {
    mParam1 = getString(ARG_PARAM1)
    mParam2 = getString(ARG_PARAM2)
}
```

See http://beust.com/weblog/2015/10/30/exploring-the-kotlin-standard-library/ for more details.